### PR TITLE
发货人组件基本使用

### PR DESCRIPTION
**Pull request by HhQr**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by SubinY** [comment]
This branch has conflicts that must be resolved

File: src/custom-components/shipper-select/shipper-select.component.ts [hunk 1]
```diff
@@ -0,0 +1,200 @@
+import { Component, OnInit, Input, forwardRef, NgModule, TemplateRef, Output, EventEmitter, ViewChild, ViewContainerRef, ElementRef, ContentChild } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR, FormsModule } from "@angular/forms";
+import { CommonModule, NgForOfContext } from "@angular/common";
+import { NgZorroAntdModule } from '../../../index.showcase';
+import { API } from '../services/api';
+import { Observable } from "rxjs/Rx";
+import { Subject } from 'rxjs/Rx';
```
Comment (giscafer) [inline src/custom-components/shipper-select/shipper-select.component.ts:7]: 这种导入合并一下，`import { Observable,Subject  } from "rxjs/Rx";`

File: src/custom-components/shipper-select/shipper-select.component.ts [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+export interface Shipper {
```
Comment (giscafer) [inline src/custom-components/shipper-select/shipper-select.component.ts:9]: 接口定义，属性请用分号`;`间隔，最后一位属性也是分号结束，参考下边

```ts
export interface CalendarOriginal {
  time: number;
  date: Date;
  year: number;
  month: number;
  firstWeek: number;
  howManyDays: number;
}
```

另外，如果**仅**是组件内部使用的`interface` ，建议不要用export导出，如果是外部公用，多组件都使用的`interface`，则建议外部定义好，组件内`import` 复用

**Review by giscafer — COMMENTED** [review]
规范注意

**Comment by giscafer** [comment]
@SubinY @HhQr  组件encapsulation 建议统一为：  encapsulation: ViewEncapsulation.None,

方便后期全局样式复写组件样式

**Pull request merged**